### PR TITLE
Allow Ledger support for Dock

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh <lovesh.bond@gmail.com>

### CODE REVIEW

File: packages/apps-config/src/api/constants.ts [hunk 1]
```diff
@@ -20,3 +20,5 @@ export const POLKADOT_GENESIS = getGenesis('polkadot');
 export const POLKADOT_DENOM_BLOCK = new BN(1248328);
 
 export const KULUPU_GENESIS = getGenesis('kulupu');
+
+export const DOCK_GENESIS = '0xf73467c6544aa68df2ee546b135f955c46b90fa627e9b5d7935f41061bb8a5a9';
```
Comment (jacogr) [inline packages/apps-config/src/api/constants.ts:24]: It is available, see https://github.com/polkadot-js/common/blob/master/packages/networks/src/index.ts#L229 

We don't want o maintain the chain-specific stuff here, i.e. just get it from the same route as above. (The networks package determines what is available for ss58/genesis here and in the extension)

TL;DR `getGenesis('dock-mainnet')`

Comment (lovesh) [reply packages/apps-config/src/api/constants.ts]: Thanks for reminding me. Fixed.

**Review by jacogr — APPROVED** [review]
Thank you!
